### PR TITLE
fix: expose load_external_data for large models

### LINKED ISSUE #138: [Question] Converting models over 2GB
Thank you for the excellent work!

When using convert_model command in ghcr.io/els-rd/transformer-deploy:0.5.1 to convert models over 2GB, the following error occurs.

```
[09/13/2022-08:29:13] [TRT] [E] onnx2trt_utils.cpp:741: Size mismatch when importing initializer: roberta.embeddings.word_embeddings.weight. Expected size: 0 , actual size: 1024008192
```

Though editing following line to set `load_external_data=True` solves this error, is it right workaround?

https://github.com/ELS-RD/transformer-deploy/blob/cc781dbe925cccdc309e0a96501dc20b979b4627/src/transformer_deploy/backends/pytorch_utils.py#L168

Thanks,

### PR DESCRIPTION
**Pull request by ayoub-louati**
Tested on cloud-dev with roberta-large.
closes #138

### CODE REVIEW

File: src/transformer_deploy/utils/args.py [hunk 1]
```diff
@@ -45,6 +45,13 @@ def parse_args(commands: List[str] = None) -> argparse.Namespace:
             "For private models, use `True` to use local cached token, or a string of your HF API token"
         ),
     )
+    parser.add_argument(
+        "-l",
```
Comment (gaetansnl) [inline src/transformer_deploy/utils/args.py:49]: I'm not sure it's a good idea to use short flags like `-l` for such specific command. Is it possible to only have `--load-external-data` ?

**Review by gaetansnl — CHANGES_REQUESTED** [review]
I found a problem I think. And could you add an example command to test the behavior in the PR comments ?

File: src/transformer_deploy/utils/args.py (hunk 1 above)
Comment (gaetansnl) [reply src/transformer_deploy/utils/args.py:49]: https://docs.python.org/3/library/argparse.html#action

Comment (ayoub-louati) [reply src/transformer_deploy/utils/args.py:49]: @gaetansnl argparse.BooleanOptionalAction works only with python3.9 or higher and I think we did not impose this versions for transformer-deploy project.

File: src/transformer_deploy/backends/pytorch_utils.py [hunk 2]
```diff
@@ -165,7 +166,7 @@ def convert_to_onnx(
             training=TrainingMode.EVAL,  # always put the model in evaluation mode
             verbose=False,
         )
-    proto = onnx.load(output_path, load_external_data=False)
```
[the remark this answers is not in the document]
Comment (ayoub-louati) [reply src/transformer_deploy/backends/pytorch_utils.py:168]: I asked Michael and he said that he wanted to work and do modifications only on graphs without having to load the whole model.

**Comment by ayoub-louati** [comment]
To test you can just run the convert_model command with the given parameters, for example (in local):
convert_model -m roberta-large --backend onnx --seq-len 16 256 256 --task embedding --load-external-data

File: src/transformer_deploy/utils/args.py (hunk 1 above)
Comment (gaetansnl) [reply src/transformer_deploy/utils/args.py:49]: ok so we can implement the flags manually

**Review by gaetansnl — APPROVED** [review]
tested with deepset/xlm-roberta-large-squad2

**Pull request merged**